ppdet/data/transform/operators.py

```diff
@@ -1341,7 +1378,9 @@ def __call__(self, sample, context=None):
         sample['image'] = canvas
         if 'gt_bbox' in sample and len(sample['gt_bbox']) > 0:
             sample['gt_bbox'] += np.array([x, y] * 2, dtype=np.float32)
-
+        if len(sample['gt_poly']) != 0 and None not in sample['gt_poly']:
```

Comment:
why `None not in sample['gt_poly']`？ should here be `'gt_poly' not in sample`?

Reply:
thx. fix to: if 'gt_poly' in sample and len(sample['gt_poly']) > 0: